content/en/infrastructure/resource_catalog/governance/_index.md

```diff
@@ -0,0 +1,86 @@
+
+Specifically, you can:
+
+- Define a [custom policy](#create-a-custom-policy), which involves choosing a resource type, the attribute on the resource type, and target value(s) the attribute should have
+- Define a [tagging policy](#create-a-tagging-policy), which involves a resource type and the desired tag key and value the resource type should have
+- Access a dedicated view for each policy where you can see its list of non-compliant resources and compliance score
+- Filter, group, and export the list of non-compliant resources for a policy so you can prioritize and assign the work
+
+## Create a custom policy
+
+{{< img src="/infrastructure/resource_catalog/governance/custom-policy-example-1.png" alt="A custom policy reflecting a compliance score of thirty percent." width="100%">}}
+
+Custom policies require specific values in your cloud resource attributes within Datadog based on your organization's infrastructure best practices.
+
+To create a custom policy:
+
+1. Navigate to **Infrastructure > Resource Catalog** and click the [**Governance**][1] tab.
+2. Click the **New Custom Policy** button.
+3. Select a Resource Type from the dropdown menu.
+4. Optionally, search for additional dataset filters.
+5. Select a target resource attribute and desired value.
+6. Optionally, add instructions for remediation.
+7. A name is automatically generated based on the data entered, but you can modify it.
+8. Click **Create Custom Policy**.
+
+By clicking the new policy, you can review all non-compliant resources and filter them by region, environment, account, service, or team. You can also group them by attributes or tags.
+
+## Create a tagging policy
+
+Tagging policies require specific tag keys and tag value formats on your infrastructure resources across Datadog.
+
+To create a tagging policy:
+
+1. Navigate to **Infrastructure > Resource Catalog** and click the [**Governance**][1] tab.
+2. Click the **New Tagging Policy** button.
+3. Choose the resource types the policy applies to.
+4. Define the required tag key and its allowed values.
+5. A name is automatically generated based on the data entered, but you can modify it.
+6. Click **Create Tagging Policy**.
+
+By clicking the new policy, you can review all non-compliant resources and filter them by cloud, region, environment, account, service, team, or tag. You can also group them by attributes or tags.
+
+[1]: https://app.datadoghq.com/infrastructure/catalog/governance
+
+## Updating policies
+
+To update a policy:
+
+1. Click the policy.
+2. Click the **Edit** button.
+3. Modify the policy as needed.
+4. Click **Update Custom Policy** or **Update Tagging Policy**.
```

Comment:
This seems like a pretty simple workflow that might not need a full stepped procedure. I think it'll be clear to the user that they need to basically save their changes. Similar to the deleting and exporting instructions below:
```suggestion
To update a policy, click the policy, then click the **Edit** button and modify as needed.
```